src/android/Capture.java

```diff
@@ -332,8 +332,8 @@ private void captureVideo(Request req) {
     public void onActivityResult(int requestCode, int resultCode, final Intent intent) {
         final Request req = pendingRequests.get(requestCode);
 
-        // Result received okay
-        if (resultCode == Activity.RESULT_OK) {
+        // Result received okay or the capture image intent has been cancelled
+        if (resultCode == Activity.RESULT_OK || (req.action == CAPTURE_IMAGE && intent == null)) {
```

Comment:
Why specifically `CAPTURE_IMAGE`? It may be the case for audio and video capture too?